modules/modifying-kubelet-as-one-time-scenario.adoc

```diff
@@ -0,0 +1,94 @@
+[id="modifying-kubelet-one-time_{context}"]
+= Modifying the kubelet as a one-time scenario
+
+.Procedure
+
+To modify the kubelet in a one-time scenario without rebooting the node due to the change of `machine-config(spec":{"paused":false}})`, allowing you to modify the kubelet without affecting the service, follow this procedure.
+
+. Connect to the node in debug mode:
++
+[source,terminal]
+----
+$ oc debug node/<node>
+----
++
+[source,terminal]
+----
+$ chroot /host
+----
+
+. Once access is established, check the content:
++
+[source,terminal]
+----
+$ systemctl cat kubelet
+----
++
+.Example output
+[source,terminal]
+----
+# /etc/systemd/system/kubelet.service
+[Unit]
+Description=Kubernetes Kubelet
+Wants=rpc-statd.service crio.service
+After=crio.service
+
+[Service]
+Type=notify
+ExecStartPre=/bin/mkdir --parents /etc/kubernetes/manifests
+ExecStartPre=/bin/rm -f /var/lib/kubelet/cpu_manager_state
+EnvironmentFile=/etc/os-release
+EnvironmentFile=-/etc/kubernetes/kubelet-workaround
+EnvironmentFile=-/etc/kubernetes/kubelet-env
+
+ExecStart=/usr/bin/hyperkube \
+kubelet \
+  --config=/etc/kubernetes/kubelet.conf \
+  --bootstrap-kubeconfig=/etc/kubernetes/kubeconfig \
+  --rotate-certificates \
+  --kubeconfig=/var/lib/kubelet/kubeconfig \
+  --container-runtime=remote \
+  --container-runtime-endpoint=/var/run/crio/crio.sock \
+  --allow-privileged \
+  --node-labels=node-role.kubernetes.io/master,node.openshift.io/os_id=${ID} \
+  --minimum-container-ttl-duration=6m0s \
+  --client-ca-file=/etc/kubernetes/ca.crt \
+  --cloud-provider= \
+  --volume-plugin-dir=/etc/kubernetes/kubelet-plugins/volume/exec \
+  \
+  --anonymous-auth=false \
+  --v=1 \
+
+Restart=always
+RestartSec=10
+
+[Install]
+WantedBy=multi-user.target
+# /etc/systemd/system/kubelet.service.d/10-default-env.conf
+----
+
+. Define the new verbosity required in `/etc/systemd/system/kubelet.service` unit. In this example, it is changed from `v=1` to `v=8`:
```

Comment:
This is outdated... The file is located `/etc/systemd/system/kubelet.service.d/20-logging.conf` and looks like:
https://github.com/openshift/machine-config-operator/blob/master/templates/common/_base/files/kubelet-log-level.yaml

Creating a 99-logging.conf file should override the default value.

Reply:
Creating a new file is probably preferred, since the MCD will detect changes in the managed 20-logging.conf file and mark the node as degraded.

Reply:
@rphillips Thanks! I made an update. PTAL